src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = (props) => {
```

Comment:
Nice work destructuring your props, though you could eliminate line 4 by doing this:
```suggestion
const NumberButton = ({number}) => {
```